### PR TITLE
Update _config.yml

### PR DESCRIPTION
**Pull request by Clalloures**

### CODE REVIEW

**Comment by Clalloures** [comment]
yes

File: _config.yml [hunk 1]
```diff
@@ -1,15 +1,16 @@
-title: Your awesome title
-author: GitHub User
-email: your-email@domain.com
+title: My Profile
+author: Clarissa Lima
+email: clarissalloures@gmail.com
 description: > # this means to ignore newlines until "baseurl:"
-  Write an awesome description for your new site here. You can edit this
```
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml [hunk 2]
```diff
@@ -1,15 +1,16 @@
-title: Your awesome title
-author: GitHub User
-email: your-email@domain.com
+title: My Profile
```
Comment (Clalloures) [inline _config.yml]: pk

File: _config.yml [hunk 3]
```diff
@@ -1,15 +1,16 @@
-title: Your awesome title
```
Comment (Clalloures) [inline _config.yml]: ok

Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
-author: GitHub User
```
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml [hunk 5]
```diff
@@ -1,15 +1,16 @@
...
-email: your-email@domain.com
```
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml (hunk 1 above)
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml [hunk 6]
```diff
@@ -1,15 +1,16 @@
...
+title: My Profile
+author: Clarissa Lima
+email: clarissalloures@gmail.com
 description: > # this means to ignore newlines until "baseurl:"
-  Write an awesome description for your new site here. You can edit this
-  line in _config.yml. It will appear in your document head meta (for
```
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml [hunk 7]
```diff
@@ -1,15 +1,16 @@
...
-  Google search results) and in your feed.xml site description.
```
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml [hunk 8]
```diff
@@ -1,15 +1,16 @@
...
-
```
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml [hunk 9]
```diff
@@ -1,15 +1,16 @@
...
+  This is my profile :)
 # social links
-twitter_username: your-twitter-handle
```
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml [hunk 10]
```diff
@@ -1,15 +1,16 @@
...
-github_username:  your-github-handle
```
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml [hunk 11]
```diff
@@ -1,15 +1,16 @@
...
+github_username:  Clalloures
 
 show_excerpts: true # set to false to remove excerpts on the homepage
 
-theme: jekyll-theme-cayman
```
Comment (Clalloures) [inline _config.yml]: ok

File: _config.yml (hunk 3 above)
Comment (mirianfsilva) [inline _config.yml]: add " "

**Pull request merged**